templates/microprofile-payara/files/README.md

```diff
@@ -0,0 +1,20 @@
+# Payara Micro Maven template for Platform.sh
```

Comment:
You say MicroProfile on all the other one's. Any reason it's just "Micro" here?

Reply:
That is the product name: 
https://www.payara.fish/software/payara-server/payara-micro/